### PR TITLE
[stable/grafana] Ability to give a subpath for an extra secret

### PR DESCRIPTION
**Pull request by audig**
@zanhsieh @rtluckie @maorfr 
#### What this PR does / why we need it:
Add the ability to give a subpath for an extra secret.
Use case:
We use a secret with multiple data and we don't want to override the directory /etc/ssl/certs/ but just add a file. SubPath in volumeMount help to achive this.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x ] Chart Version bumped
- [x ] Variables are documented in the README.md
- [x ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.0.14
+version: 5.0.15
```
Comment (zanhsieh) [inline stable/grafana/Chart.yaml]: Should be 5.0.18.

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: stable/grafana/Chart.yaml (hunk 1 above)
Comment (audig) [reply stable/grafana/Chart.yaml]: Ok, but can you explain why to 5.0.18 and not 5.0.17 next version of the chart ?

**Comment by audig** [comment]
@zanhsieh 
Sorry but it's the first time I commit in a github with sign.
I try to sign the commit where i bump the version but i can't. I've some problems to amend my commit.

**Comment by audig** [comment]
Sorry but i reopen a new PR because i break some commits and didn't know how to revert all of this

**Pull request closed without merging**